openshift-knative-operator/pkg/common/images.go

```diff
@@ -0,0 +1,34 @@
+package common
+
+import (
+	"strings"
+)
+
+const imagePrefix = "IMAGE_"
+
+// ImageMapFromEnvironment generates a map of deployment/container to image from
+// the passed environment variables.
+func ImageMapFromEnvironment(env []string) map[string]string {
```

Comment:
wasn't there some "logic" like this before, somewhere?

Reply:
yeah, in the OSDK based operator and in the bandaid PRs I sent towards the operator forks. Mostly 1:1 copies with little touchups here and there.

Reply:
that's cool with me, @markusthoemmes